spynnaker/pyNN/models/neuron/abstract_population_vertex.py

```diff
@@ -303,6 +322,31 @@ def set_max_atoms_per_dimension_per_core(self, new_value):
     def set_structure(self, structure):
         self.__structure = structure
 
+    @property
+    def single_core_capable(self):
+        """ Determine if the vertex can manage to operate on a single core,
+            or if a split synapse-core is more appropriate.
+
+            Note that this is currently based only on the ITCM available, not
+            on the incoming synapses.
+
+        :rtype: bool
+        """
+        if self.__synapse_dynamics is None:
+            return True
+        return self.__synapse_dynamics.is_single_core_capable
+
+    @property
+    def n_synapse_cores_required(self):
+        """ Get the estimated number of synapse cores required.
+
+            Note this is currently hard-coded but could be updated to work
+            this out based on the number of incoming synapses
+
+        :rtype: int
+        """
+        return 1
```

Comment:
Is above method(currently single_core_capable) is True while is even 1 synapse cores required?

Reply:
I have updated the comment to make it more clear I hope!  This property is only requested if the one above returns False.